modules/dev-tools/scripts/lint.sh

```diff
@@ -39,19 +39,25 @@ case $MODE in
   "pre-commit")
     print_yellow "Running prettier & eslint on changed files..."
 
+    NAME_PATTERN=`echo "^$DIRECTORIES/.*\.js$" | sed -e 's/,/|/g' | sed -e 's/{/(/g' | sed -e 's/}/)/g'`
```

Comment:
Curious why you have to do this:
`sed -e 's/,/|/g' | sed -e 's/{/(/g' | sed -e 's/}/)/g'`

Reply:
I'm basically translating the glob pattern to a regex pattern.